src/proxy/server.rs

```diff
@@ -95,11 +95,19 @@ pub struct NoOriginalDst;
 impl Source {
     pub fn orig_dst_if_not_local(&self) -> Option<SocketAddr> {
         match self.orig_dst {
-            None => None,
+            None => {
+                trace!("no SO_ORIGINAL_DST on source");
```

Comment:
The `trace!`s in this file are not needed, but I found them helpful when testing and therefore probably a good reason to keep around.